moveos/moveos-stdlib/moveos-stdlib/sources/raw_table.move

```diff
@@ -19,8 +19,10 @@ module moveos_std::raw_table {
     const ErrorAlreadyExists: u64 = 1;
     /// Can not found the key in the table
     const ErrorNotFound: u64 = 2;
+    /// Duplicate operation on the table
+    const ErrorDuplicateOperation: u64 = 3;
```

Comment:
These errors are different from the raw_table native implemention.

Reply:
It is better to define it and make sure the error code in Rust is consistent with error code in Move.